tkp/accessors/fitsimageblob.py

```diff
@@ -0,0 +1,53 @@
+import numpy
+import logging
+from tkp.accessors.fitsimage import FitsImage
+
+logger = logging.getLogger(__name__)
+
+
+class FitsImageBlob(FitsImage):
```

Comment:
This looks fine, but to be picky, there's quite a lot of repetition here. A nice refinement might be to refactor FitsImage to accept an hdulist, and then subclass it with a couple of extra lines to deal with the AARTFAAC specific URL functionality.